script/release/notes/notes.js

```diff
@@ -21,6 +21,8 @@ const TROP_LOGIN = 'trop[bot]';
 
 const NO_NOTES = 'No notes';
 
+const ignoreNotes = new Set(['none', 'no notes', 'notes']);
```

Comment:
We should copy the logic used to designate no-notes in Clerk: 2e264524f9afd34c691fea/src/note-utils.ts#L41-L52

Reply:
Ok, I'll close this PR then and make a modification to that clerk file instead. Thanks!